micro-ui/web/micro-ui-internals/packages/modules/core/CHANGELOG.md

```
@@ -1,4 +1,6 @@
 # Changelog 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)